### PR TITLE
Add status registration message

### PR DESCRIPTION
**Pull request by jesusbv**
- Show registration status in console

This Fixes bsc#1197113

### CODE REVIEW

File: usr/sbin/registercloudguest [hunk 1]
```diff
@@ -417,6 +419,7 @@ while not base_registered:
         stderr=subprocess.PIPE)
     res = p.communicate()
     if p.returncode != 0:
+        print('Registration failed, see /var/log/guestregister for details')
```
Comment (rjschwei) [inline usr/sbin/registercloudguest:422]: We do not want to print the failure message for every update server we are trying, one summary message, you'll have to set a flag.

File: usr/sbin/registercloudguest [hunk 2]
```diff
@@ -79,9 +79,11 @@ def register_modules(extensions, products, registered=[]):
             )
             res = p.communicate()
             if p.returncode != 0:
+                print('Registration failed, see /var/log/guestregister for details')
```
Comment (rjschwei) [inline usr/sbin/registercloudguest:82]: Ditto, one summary message if registration succeeds or fails.

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: usr/sbin/registercloudguest (hunk 2 above)
Comment (jesusbv) [reply usr/sbin/registercloudguest]: Ack, done.

File: usr/sbin/registercloudguest [hunk 3]
```diff
@@ -484,6 +488,11 @@ prod_data = json.loads(res.text)
 extensions = prod_data.get('extensions')
 register_modules(extensions, products)
 
+if registration_returncode != 0:
+    print('Registration failed, see /var/log/guestregister for details')
```
Comment (rjschwei) [inline usr/sbin/registercloudguest:492]: This is an error and needs to go to stderr, i.e. `file=sys.stderr'

exit code is set later in the code.

**Review by rjschwei — CHANGES_REQUESTED** [review]

**Review by rjschwei — APPROVED** [review]

**Pull request merged**